### ISSUE TITLE
new rule: react-a11y-anchors

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#199]
Link text should be as least four 4 characters long. (http://oaa-accessibility.org/wcag20/rule/38/)

Links with the same HREF should have the same link text. (http://oaa-accessibility.org/wcag20/rule/39/)

Links that point to different HREFs should have different link text. (http://oaa-accessibility.org/wcag20/rule/41/)

Links with images and text content, the alt attribute should be unique to the text content or empty. (http://oaa-accessibility.org/wcag20/rule/43/)

Enforce an anchor element's href prop value is not just #.

From HTML Code Sniffer SC 4.1.2: Name, Role, Value (Level A) (
SC 4.1.2: Name, Role, Value (Level A))
	• http://www.w3.org/TR/WCAG20-TECHS/H91
	• Anchor element found with an ID but without a href or link text. 
	• Anchor element found with a name attribute but without a href or link text
	• Anchor element found with no link content and no name and/or ID attribute
	• Anchor element with no href, but with an ID and text. This breaks the intended role of an "A" element being a link. It will usually be around text inside another parent element (such as a heading); the ID should be moved up for the same effect.
	• Link with text, but no href or ID. This means it does not have an accessibility API "value"
Link with a href attribute, but without link text. This means it does not have an accessibility API "name".

Notes: 
"Links with the same HREF should have the same link text." You could keep as a static variable across all the files a map from link to link text and enforce this across files.
